src/bot/commands/UnignoreCommand.ts

```diff
@@ -9,7 +9,9 @@ export default class UnignoreCommand implements Command {
   public readonly USAGE = 'Usage: !unignore <user>';
 
   public run(message: Message) {
-    if (!message.member.hasPermission(Permissions.FLAGS.ADMINISTRATOR!)) return;
+    if (!message.member || !message.member.hasPermission(Permissions.FLAGS.ADMINISTRATOR!)) {
```

Comment:
We can also do `if (!message.member?.hasPermission(...))` because we are on TypeScript 3.7 🎉

Whatever you prefer to be honest. I think I like the more explicit version more.